### ISSUE TITLE
Server side copy - Box - item_name_in_use 409 error with existing files

### ISSUE THREAD

**Issue by a6** [rclone/rclone#3511]
Linked from [forum posting](https://forum.rclone.org/t/server-side-copy-box-item-name-in-use-409-error-with-existing-files/11365).

If I attempt a server side copy from one box path to another, any file existing in both paths that has been updated in the src path will not be updated in the dst path. Instead, an error is thrown:

    DEBUG : : Sizes differ (src 32 vs dst 25)
    ERROR : : Failed to copy: Error "item_name_in_use" (409): Item with the same name already exists

The command is something like:

    rclone copy boxRemote:path1 boxRemote:path2

where path2 is intended to have a copy of all files that exist in path1.

The first time this copy is called, everything works because there are no existing files of path1 in path2. However, if later, a file is updated in path1 and then the above copy command is attempted, this `item_name_in_use` error is thrown.

This is with rclone 1.48 on Ubuntu x64 16.04.

I suspect this is not desired behavior, because the same types of copy commands work just fine if src is local instead of a box remote.

**Comment by ncw** [comment]
I reproduced this like this with the latest beta

```
$ date -Is | rclone rcat boxcom:test1/file1.txt
$ date -Is | rclone rcat boxcom:test2/file1.txt
$ rclone copy -vv --retries 1 --low-level-retries 1 boxcom:test2 boxcom:test1
2019/09/05 13:03:01 DEBUG : rclone: Version "v1.49.0-024-g5dad88ae-fix-3506-names-beta" starting with parameters ["rclone" "copy" "-vv" "--retries" "1" "--low-level-retries" "1" "boxcom:test2" "boxcom:test1"]
2019/09/05 13:03:01 DEBUG : Using config file from "/home/ncw/.rclone.conf"
2019/09/05 13:03:02 DEBUG : file1.txt: Modification times differ by -8s: 2019-09-05 05:02:38 -0700 -0700, 2019-09-05 05:02:30 -0700 -0700
2019/09/05 13:03:02 INFO  : box root 'test1': Waiting for checks to finish
2019/09/05 13:03:02 DEBUG : file1.txt: SHA-1 = b28a9df4b94db8bc1fb4ad48a3b0137d8d6f3348 (box root 'test2')
2019/09/05 13:03:02 DEBUG : file1.txt: SHA-1 = f261a573e4a89950c5e8d5d22367570274a987d8 (box root 'test1')
2019/09/05 13:03:02 DEBUG : file1.txt: SHA-1 differ
2019/09/05 13:03:02 INFO  : box root 'test1': Waiting for transfers to finish
2019/09/05 13:03:02 ERROR : file1.txt: Failed to copy: Error "item_name_in_use" (409): Item with the same name already exists
2019/09/05 13:03:02 ERROR : Attempt 1/1 failed with 3 errors and: Error "item_name_in_use" (409): Item with the same name already exists
2019/09/05 13:03:02 Failed to copy with 3 errors: last error was: Error "item_name_in_use" (409): Item with the same name already exists
```

I suspect we need to add an integration test for
- Copy over existing
- Move over existing

and fix any backends that have a problem starting with box!

**Comment by antonio-petricca** [comment]
Same problem for me.

**Comment by doodooface** [comment]
Same problem as well. I just end up rclone copying/moving to a new temp folder

**Comment by ncw** [comment]
This is merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.69 - thanks @nielash
